readme.md

```diff
@@ -167,171 +180,158 @@ console.log(ulChildren);
-  console.log("you clicked the h1!")
-})
-var anchor = document.querySelector("a")
-anchor.addEventListener("click", function(){
-  console.log("you clicked the anchor!")
-})
+```javascript
+$( "ul" ).empty();
 ```
 
-> The first argument for `addEventListener` is the event you are listening for. In this case, the `click` event. The second argument is the thing that should happen when the event occurs, otherwise known as a callback. There's much to learn about `events` and `callbacks`. So much so that we have devoted another entire class to it.. So more to follow!
 
-Hmm, if we click on the link, its still going to google. What gives? Turns out anchors have a default action when we click them. They go to the specified link. We need a way to ignore the default behavior. We can do this by leveraging the event object.
+## I Do: Putting it all together (5 minutes)
+
+I'm going to use jQuery methods to traverse the DOM and turn the text contained in the first list item of an unordered list red. Methods are just functions that are properties of an object.
+
+`script.js`
 
 ```js
-var anchor = document.querySelector("a")
-anchor.addEventListener("click", function(evt){
-  evt.preventDefault()
-  console.log("you clicked the anchor!")
-})
+$(document).ready(function(){
+
+// selected the ul tag
+// .children alone selects all child elements
+// using .eq reduces the set to a specific index
+// the .css method set the color of the text
+$("ul").children().eq(0).css( "color", "red" );
+
+});
 ```
 
-> There's lots more about events that you'll learn in the next classes. For now, know that every event listener you add allows us the access the event object as an argument of the function callback.
+## You Do: Selecting DOM elements (20 min)
 
-## Exercises
+Go to [this repo](https://github.com/ga-wdi-exercises/js-dom-quotes) and follow the instructions in the jQuery branch.
 
-- [Take the Money and Run](https://github.com/ga-wdi-exercises/ttmar)
-- [color scheme switcher](https://github.com/ga-wdi-exercises/color-scheme-switcher)
+Test out grabbing DOM elements using the selectors above.
 
 
-## Homework
-Finish [Take the Money and Run](https://github.com/ga-wdi-exercises/ttmar)
+## You do: Logo hijack (15 min)
 
+1. Open up www.google.com in Chrome or Firefox, and open up the console.
+- find an image url for the yahoo logo
+- Store the url to the Yahoo logo in a variable.
+- Find the Google logo using JS and store it in a variable.
+- Modify the source of the logo IMG so that it's a Yahoo logo instead.
+- Find the Google search button and store it in a variable.
+- Modify the text of the button so that it says "Yahooo!" instead.
 
-## References
-- https://developer.mozilla.org/en-US/docs/Web/API/Event
+Bonus: Add a new element between the image and the search textbox, telling the world that "Yahoo is the new Google".
 
 [![Build Status](https://travis-ci.org/ga-wdi-lessons/js-dom.svg?branch=master)](https://travis-ci.org/ga-wdi-lessons/js-dom)
```

Comment:
this is a steep ask, but you can do this tomorrow during james' lesson, but putting in all the vanilla JS equivalences as details